Dockerfile

```diff
@@ -77,6 +77,11 @@ RUN case ${TARGETPLATFORM} in \
     esac && \
     /opt/conda/bin/conda clean -ya
 RUN /opt/conda/bin/pip install torchelastic
+RUN IS_CUDA=$(python -c 'import torch ; print(torch.cuda._is_compiled())'); \
```

Comment:
What if user wants to build CPU docker version? Should we somehow map IS_CUDA to say `INSTALL_CHANNEL` to figure out whether we want `cu` or `cpu` versions?

Reply:
Done. Added check 
```
if test "${IS_CUDA}" != "True" -a ! -z "${CUDA_VERSION}"; then \
        exit 1; \
fi
```